core/src/test/java/org/elasticsearch/search/aggregations/bucket/SignificantTermsSignificanceScoreIT.java

```diff
@@ -273,12 +272,15 @@ public void testXContentResponse() throws Exception {
         }
 
         XContentBuilder responseBuilder = XContentFactory.jsonBuilder();
+        responseBuilder.startObject();
         classes.toXContent(responseBuilder, null);
+        responseBuilder.endObject();
```

Comment:
It's possible here but `ScriptSortBuilderTests.testToXContentWithParams()` uses custom `ToXContentParams` and `Strings.toString(XContentBuilder, true)` forces to use empty params.